Tests/Marketplace/marketplace_services.py

```diff
@@ -955,7 +963,19 @@ def upload_to_storage(self, zip_pack_path, latest_version, storage_bucket, overr
 
             with open(zip_pack_path, "rb") as pack_zip:
                 blob.upload_from_file(pack_zip)
+
             if private_content:
+                print('entering private content')
```

Comment:
```suggestion
                logging.debug('entering private content')
```
Better to use logging than printing